spec/controllers/pages_controller_spec.rb

```diff
@@ -94,28 +94,30 @@
     end
   end
 
-  describe 'GET #toggle_locale' do
+  describe 'POST #toggle_locale' do
     context 'When user is signed in' do
       let(:user) { create(:user) }
```

Comment:
I know this was here previously, but do we need to seed a user here? Can me mock it out using `build` as opposed to `create` so we're not hitting the db?